### ISSUE TITLE
[Bug]: The type or namespace name 'Default' does not exist in the namespace 'OneSignal'

### ISSUE THREAD

**Issue by NamCkiku** [OneSignal/OneSignal-DotNet-SDK#26]
### What happened?

net7.0;net7.0-android;net7.0-ios
Buid output error:
The type or namespace name 'Default' does not exist in the namespace 'OneSignal' (are you missing an assembly reference?

![image](https://user-images.githubusercontent.com/15065058/209113509-045313b3-745f-4dbc-9e19-a4609baed6bf.png)

### Steps to reproduce?

```Markdown
1.Target net7.0;net7.0-android;net7.0-ios
2.Register OneSignal :
3.OneSignal.Default.Initialize(oneSignalKey)
4.Rebuild
5.Rebuild Error
```

### What did you expect to happen?

Fix bug 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by brismithers** [comment]
sorry for the delay @NamCkiku .  Out of curiosity, does it build when you fully qualify the class i.e. `OneSignalSDK.DotNet.OneSignal.Default.Initialize(oneSignalKey)` ?

**Comment by dave-u** [comment]
> sorry for the delay @NamCkiku . Out of curiosity, does it build when you fully qualify the class i.e. `OneSignalSDK.DotNet.OneSignal.Default.Initialize(oneSignalKey)` ?

I've had this same issue and have now fully qualified the class as you've suggested and can confirm it now builds successfully.

**Comment by brismithers** [comment]
thank you for confirming, this is a namespacing issue that will need a code change.  Marking this as a bug to be resolved.

**Comment by brismithers** [comment]
This fix has been released in [4.3.2](https://github.com/OneSignal/onesignal-dotnet-sdk/releases/tag/4.3.2)
